src/PrestaShopBundle/Routing/LegacyControllerConstants.php

```diff
@@ -1,4 +1,29 @@
 <?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
```

Comment:
Whis this license is different from the other one in modules/index.php?

Reply:
The core code is on OSL license, the modules use AFL

Reply:
The script simply changed the index file in the modules folder, which makes sense But honestly it's just a safety index file we don't care about its license But since the script prefers it in AFL at least we won't have a new update next time the script is run